### ISSUE TITLE
Some CNAME items in popup not in blue

### ISSUE THREAD

**Issue by mergitrack** [uBlockOrigin/uBlock-issues#921]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

Some items in the popup that have CNAME are not display in blue.

### A specific URL where the issue occurs

https://github.com/gorhill/uBlock/releases
https://en.wikipedia.org/wiki/Main_Page
https://www.instagram.com/

### Steps to Reproduce

1. Enter one of the URLs from above. 
2. Click the uBlock button in Firefox to display the popup window. 
3. Get the URL address. 
4. Check https://mxtoolbox.com/SuperTool.aspx and lookup CNAME for the URL: 
- avatars1.githubusercontent.com  shows CNAME as github.map.fastly.net (but not shown in blue in popup)
- avatars2.githubusercontent.com  showns CNAME as github.map.fastly.net  (but not shown in blue in popup)
- en.wikipedia.org  shows CNAME as dyna.wikimedia.org (but not shown in blue in popup) -- does show in blue in the Logger
- www.instagram.com  shows CNAME as z-p42-instagram.c10r.facebook.com (but not shown in blue in popup) -- does show in blue in the Logger

### Expected behavior:

These should be displayed in blue in the popup if they are CNAME items. 

### Actual behavior:

Items in the popup are not in blue -- but may be in blue in the Logger. 

### Your environment

* uBlock Origin version:  1.25.0
* Browser Name and version: Firefox 73.0.1 (64-bit)
* Operating System and version:  Windows 10 Home 1903

**Comment by uBlock-user** [comment]
`avatars1.githubusercontent.com` is not the CNAME, hence not in blue. Only CNAME are shown in blue, because they're not network requests made by the document.

**Comment by mergitrack** [comment]
Can you confirm or clarify if dyna.wikimedia.org should be in blue for en.wikipedia.org, and z-p42-instagram.c10r.facebook.com be in blue for www.instagram.com?  Thanks.

**Comment by uBlock-user** [comment]
Can't reliably reproduce CNAME on those sites right now, but I can always reproduce CNAME on ghacks.net where `ssl.marfeelcdn.com` always appears in blue in the popup panel and in the logger.

**Comment by gorhill** [comment]
I see the canonical names reported in the popup panel:

![a](https://user-images.githubusercontent.com/585534/75673340-b8747380-5c50-11ea-80cd-67745c1c7f34.png)

![a](https://user-images.githubusercontent.com/585534/75673378-d3df7e80-5c50-11ea-99bb-5025c09f92eb.png)

![a](https://user-images.githubusercontent.com/585534/75673431-ed80c600-5c50-11ea-8873-5c048b3b5b1a.png)

**Comment by gorhill** [comment]
To be clear, the aliases themselves are not reported in blue. The hostnames hidden behind aliiases are reported in blue.

**Comment by gwarser** [comment]
@mergitrack do you see `+all` in first row? Click on it.

**Comment by mergitrack** [comment]
I do see it when I click on the +all.  I was not aware that was clickable.  Thanks.

**Comment by gorhill** [comment]
> I do see it when I click on the +all.

`github.map.fastly.net` is a top-level domain, so you should have been able to see it without the need to expand to make subdomains visible.

**Comment by mergitrack** [comment]
Correct, I was able to see github.map.fastly.net in popup in blue without the +all.   
And clicking on the +all allowed me to see the entries in popup in blue for en.wikipedia.org and www.instagram.com.  
I was not previously aware of the function of the +all toggle.  
Thanks for the info.

**Comment by gorhill** [comment]
Originally you said:

> `avatars1.githubusercontent.com` shows CNAME as `github.map.fastly.net` (but **not shown in blue** in popup)

Just above you said:

>  I was able to see `github.map.fastly.net` in popup in blue without the +all

These two statements are contradictory. Can you clarify?

**Comment by mergitrack** [comment]
Apologies for the confusion.  I submitted several example URLs and that complicated things.  

On github.com I see one blue site regardless of the subdomain on or off.  

Shown in blue regardless of subdomain setting: 
 
github.map.fastly.net
user-images.githubusercontent.com

avatars1.githubusercontent.com and avatars2.githubusercontent.com are not shown in blue regardless of subdomain setting -- these are in black.  

----

On www.instagram.com the blue items displayed are different depending on the subdomain setting. 

With subdomain off, there is nothing in blue.  facebook.com is in black.   

With subdomain on, I see this in blue:

z-p42-instagram.c10r.facebook.com
www.instagram.com

----

The main reason I posted here was because I was expecting to see facebook.com in blue.  But it was in black for me.   

In 1.24.4, facebook.com is not listed in the popup.  

I know 1.25.0 added the CNAME function and I see facebook.com in the popup.  But I thought it should have been in blue to indicate facebook.com was from the CNAME (with subdomain off). 

But as I learned from here, I need to toggle subdomain on to see all the entries and see facebook.com in blue.  

For users that don't have subdomain expanded, going to www.instagram.com will show facebook.com in the popup in black and that doesn't provide a visual indication that it came from CNAME.  Might be something to consider to show the base domain in blue if it came from the CNAME (with the subomain toggled off).

I have a workaround for me by expanding the subdomain to see all the CNAME entries.

If it matters for this discussion, these are my rules for instagram.com:

* * 1p-script block
* * 3p-frame block
* * 3p-script block
* * inline-script block
instagram.com akamaihd.net * noop
instagram.com cdninstagram.com * noop
instagram.com facebook.com * noop
instagram.com instagram.com * noop

Attached are screenshots that might help to explain.  Showing for github.com and www.instagram.com -- with subdomain toggled off and on. 

![ub 1](https://user-images.githubusercontent.com/61667868/75729925-fa350680-5cb0-11ea-9d94-7d855ac3ac53.png)

![ub 2](https://user-images.githubusercontent.com/61667868/75729933-00c37e00-5cb1-11ea-8e0f-b29e73454461.png)

![ub 3](https://user-images.githubusercontent.com/61667868/75729940-07ea8c00-5cb1-11ea-8061-a5b589603099.png)

![ub 4](https://user-images.githubusercontent.com/61667868/75729952-0caf4000-5cb1-11ea-91a1-206860736b1c.png)
